index.html

```diff
@@ -39,8 +39,205 @@ <h1 class="hero-title" >I’m Mariane <br>
+                            <img src="./assets/Counter.svg" alt="">
+                            Back End Dev
+                        </li>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+        </div>
+    </section>
+
+    <section class="hero hero-about ">
+        <div class="container-big">
+            <article class="hero-content separete-items-down">
+                <h1 class="hero-title" >About <br>
+                    Myself <br></h1>
+                <p class="hero-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <a href="#">LET'S CONNECT</a>
+
+                <aside class="social-media">
+                    
+                    <a href="">
+                        <img src="./assets/social media icons.svg" alt="">
+                    </a>
+                </aside>
+
+                <article class="work-button work-button-about">
+                    <button class="btn" > Get my resume </button>
+                </article>
+            </article>
+            
+        </div>
+        <article class="languajes">
+            <h2>Languages</h2>
+
+            <ul class="languajes-cont">
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 1.svg" alt="">
+                    <h3>Javascript</h3>
+                </li>
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 2.svg" alt="">
+                    <h3>HTML</h3>
+                </li>
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 3.svg" alt="">
+                    <h3>CSS</h3>
+                </li>
+            </ul>
+
+            <ul class="frameworks-cont" >
+                <li class="framework-item" >
+                    <h2>Frameworks</h2>
+                    <img src="assets/arrow-right.svg" alt="">
+                </li>
+                <li class="framework-item" >
+                    <h2>Skills</h2>
```

Comment:
- [ ] Your `Languages`, `Frameworks` and `Skills` headers are currently bolder and have a larger font size than the Figma design. Kindly fix this as well and your project should be perfect :wink: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/219581785-00666822-551f-4afe-a5fe-1e39e5041b1a.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/219581937-5fcb086d-fe5a-471b-8e50-688296f720e3.png)